src/hooks/context/useAccountsContextSetUp.ts

```diff
@@ -51,8 +54,9 @@ const useAccountsContextSetUp = () => {
   useEffect(() => {
     if (keyringPairsReady && keyringPairs.length) {
       dispatchAccount(AccountActionCreators.setAccounts(keyringPairs));
+      updateSenderAccountsInformation(dispatchAccount);
```

Comment:
Seems it's never really refreshed after the initial call or am I missing something?

Reply:
I understand it's a work-in-progress but I'd appreciate if this was tracked some place so that we can figure out what is still missing and not to forget particular things.

Reply:
> Seems it's never really refreshed after the initial call or am I missing something?

Indeed. At the moment there is no refresh. The idea is that accounts are loaded at the really beginning so we have the values in the context in this initial phase of the refactor, so then we can focus on the best refresh process.

> I understand it's a work-in-progress but I'd appreciate if this was tracked some place so that we can figure out what is still missing and not to forget particular things.

OK. I will add a comment in the code.

Reply:
Awesome, thanks! I got confused because #224 was already closed, now it makes sense cause there is a list of tasks there.

Reply:
I don't know how #224 got closed since it is clearly in progress. If i did it it was a mistake.